src/test/kotlin/org/malachite/estella/BaseIntegration.kt

```diff
@@ -350,8 +350,12 @@ class BaseIntegration {
             return try {
                 val response = restTemplate.exchange(requestEntity, String::class.java)
                 val statusCode = response.statusCode
-                val responseBody = objectMapper.readValue(response.body, Any::class.java)
-                Response(statusCode, responseBody, response.headers)
+                try {
```

Comment:
try w try? Nie lepiej zrobić jednego try'a i łapać dwa wyjątki?

Reply:
To brzmi mądrze XD

Reply:
Nie uda się, bo chcę sobie wyciągnąć `response.body`, które jest dotępne dopiero w środku `try`

Reply:
`var`

Reply:
Można varem, ale po co mam ładny blok try i stałe które są nonnullable zmieniać na zmienne przypisywane w kodzie, który rzuca wyjątki